sailfishusermanagerinterface.h

```diff
@@ -12,10 +12,20 @@
 
 #include <QString>
 #include <QDBusArgument>
+#include <QDBusError>
```

Comment:
This doesn't actually use QDBusError for anything, does it?

Reply:
Not anymore yes, removed.